exercises/practice/go-counting/.docs/instructions.md

```diff
@@ -0,0 +1,36 @@
+# Description
```

Comment:
Looking at the tests it seems the exercise requires some special output format that is not clear from the instructions. I would recommend to create an append file to clarify how the board is provided and what output format is expected for each of the functions.